src/templates/app/new_account/virtual.jsx

```diff
@@ -4,7 +4,7 @@ import FormVerificationCode from '../_includes/form_verification_code.jsx';
 
 const Virtual = () => (
     <div className='gr-12 static_full'>
-        <h1>{it.L('Create New Virtual-money Account')}</h1>
+        <h1>{it.L('Create New Demo-money Account')}</h1>
```

Comment:
Do we need to change this to `Demo Account`?

Reply:
im on it bro